### PR TITLE
Fix non-interactable elements blocking input

### LINKED ISSUE #9361: Uninteractable Formspec Elements should passthrough hovers and clicks
## Problem

Currently, labels and images prevent clicking on buttons below them

## Solutions

Labels and images should passthrough clicks

```lua
local my_formspec =
	[[
		formspec_version[3]
		size[6,6]
		button[1,1;4,4;btn;Hello]
		image[2,2;1,1;default_dirt.png]
		label[1,4;#####################]
	]]


minetest.register_on_joinplayer(function(player)
	minetest.after(0.5, function()
		minetest.show_formspec(player:get_player_name(), "ss", my_formspec)
	end)
end)
```

image

### PR DESCRIPTION
**Pull request by rubenwardy**
Fixes #9361

## To do

- [ ] Fix label[] blocking input

## How to test
```lua
local my_formspec =
	[[
		formspec_version[3]
		size[6,6]
		button[1,1;4,4;btn;Hello]
		image[2,2;1,1;default_dirt.png]
		label[1,4;#####################]
	]]


minetest.register_on_joinplayer(function(player)
	minetest.after(0.5, function()
		minetest.show_formspec(player:get_player_name(), "ss", my_formspec)
	end)
end)
```

### CODE REVIEW

**Comment by MoNTE48** [comment]
Are you sure that you cannot return the old 5.1.1 behavior without copying another three hundred code from the irrlicht engine to Minetest?

**Comment by SmallJoker** [comment]
@MoNTE48 That implies reverting the recent button, (button) styling and formspec stucture cleanup changes. This bug appeared by properly using the Irrlicht API.

**Comment by rubenwardy** [comment]
> Are you sure that you cannot return the old 5.1.1 behavior without copying another three hundred code from the irrlicht engine to Minetest?

There has been a suggestion on a way to avoid this, however, we will need to embed the entirety of Irrlicht's GUI code into Minetest in the future

**Comment by MoNTE48** [comment]
@rubenwardy I thought you were thinking of not using Irrlicht for the GUI in the future. Is there really something better for 2D?
_oh well that's offtop_

File: src/gui/guiImage.cpp [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+	out->addTexture	("Texture", Texture);
+	out->addBool	("UseAlphaChannel", UseAlphaChannel);
+	out->addColor	("Color", Color);
+	out->addBool	("ScaleImage", ScaleImage);
+	out->addRect 	("SourceRect", SourceRect);
+	out->addFloat   ("DrawBoundsX1", DrawBounds.UpperLeftCorner.X);
+	out->addFloat   ("DrawBoundsY1", DrawBounds.UpperLeftCorner.Y);
+	out->addFloat   ("DrawBoundsX2", DrawBounds.LowerRightCorner.X);
+	out->addFloat   ("DrawBoundsY2", DrawBounds.LowerRightCorner.Y);
+	out->addBool    ("DrawBackground", DrawBackground);
```
Comment (sfan5) [inline src/gui/guiImage.cpp]: We don't use any of this, so these methods should be removed or have empty body.

**Pull request closed without merging**